.azure-pipelines.yml

```diff
@@ -122,6 +124,6 @@ jobs:
         displayName: "Build & Twine check"
 
       - script: |
-          pip install dist/*.whl
+          uv pip install --system "anndata @ `ls dist/*.whl`"
```

Comment:
I think I would actually keep this one as `pip`, since this is essentially testing "can pip install this wheel".